src/CommunityToolkit.Maui.Core/Layouts/UniformItemsLayoutManager.shared.cs

```diff
@@ -23,34 +23,34 @@ public UniformItemsLayoutManager(IUniformItemsLayout uniformItemsLayout)
 	/// <summary>
 	/// Arrange children
 	/// </summary>
-	/// <param name="rectangle">Grid rectangle</param>
+	/// <param name="bounds">Grid rectangle</param>
 	/// <returns>Child size</returns>
-	public override Size ArrangeChildren(Rect rectangle)
+	public override Size ArrangeChildren(Rect bounds)
```

Comment:
Breaking Change.

This is a small change that is unlikely to impact many devs, but it does break the following code:

```cs
myUniforItemsLayoutManager.ArrangeChildren(rectangle: myRectangle);
```